res/themes/light-custom/css/_custom.scss

```diff
@@ -17,70 +17,71 @@ limitations under the License.
 //
 // --accent-color
 $accent-color: var(--accent-color);
+$accent-color-50pct: var(--accent-color-50pct);
 $button-bg-color: var(--accent-color);
 $button-link-fg-color: var(--accent-color);
 $button-primary-bg-color: var(--accent-color);
+$input-darker-fg-color: var(--accent-color);
+$input-lighter-fg-color: var(--accent-color);
 $input-valid-border-color: var(--accent-color);
-$reaction-row-button-selected-border-color: var(--accent-color);
+$muted-fg-color: var(--accent-color);
 $roomsublist-chevron-color: var(--accent-color);
+$search-placeholder-color: var(--accent-color);
 $tab-label-active-bg-color: var(--accent-color);
 $togglesw-on-color: var(--accent-color);
-$username-variant3-color: var(--accent-color);
-$accent-color-50pct: var(--accent-color-50pct); //still needs alpha at .5
+$togglesw-off-color: var(--accent-color-50pct);
 //
-// --timeline-background-color
-$authpage-body-bg-color: var(--timeline-background-color);
-$button-secondary-bg-color: var(--timeline-background-color);
-$field-focused-label-bg-color: var(--timeline-background-color);
-$lightbox-border-color: var(--timeline-background-color);
-$menu-bg-color: var(--timeline-background-color);
-$avatar-bg-color: var(--timeline-background-color);
-$message-action-bar-bg-color: var(--timeline-background-color);
-$primary-bg-color: var(--timeline-background-color);
-$roomtile-focused-bg-color: var(--timeline-background-color);
-$togglesw-ball-color: var(--timeline-background-color);
-$droptarget-bg-color: var(--timeline-background-color-50pct); //still needs alpha at .5
-$authpage-modal-bg-color: var(--timeline-background-color-50pct); //still needs alpha at .59
+// --button-color
+$composer-button-color: var(--button-color);
+$dialog-close-fg-color: var(--button-color);
+$groupheader-button-color: var(--button-color);
+$header-divider-color: var(--button-color);
+$message-action-bar-border-color: var(--button-color);
+$roomheader-button-color: var(--button-color);
+$rightpanel-button-color: var(--button-color);
+$tagpanel-button-color: var(--button-color);
 //
-// --roomlist-highlights-color
-$roomtile-selected-bg-color: var(--roomlist-highlights-color);
+// --font-family
+$font-family: var(--font-family);
```

Comment:
We need a default here for older custom themes. Does this make it fall back to the default Nunito if `--font-family` is empty?:
```
$font-family: var(--font-family, $font-family);
```